src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_6001_7000.cpp

```diff
@@ -5592,7 +5598,90 @@ GUI_TEST_CLASS_DEFINITION(test_6754) {
     GTUtilsMSAEditorSequenceArea::checkSelectedRect(os, QRect(0, 0, 1, 1));
     CHECK_SET_ERR(!l.hasErrors(), "Errors in log: " + l.getJoinedErrorString());
 }
+GUI_TEST_CLASS_DEFINITION(test_6759)
+{
+    GTLogTracer l;
+
+    //    1. Open sequence
+    //    2. Open annotation file
+    //    3. Add annotation file to sequence
+    //    4. Find splitter
+    //    5. Find scroll in splitter
+    //    6. Scroll to end, then to begin
+
+    GTFileDialog::openFileList(os,
+                               testDir + "_common_data/regression/6759/",
+                               QStringList() << "annotations.gb"
+                                             << "sequence.gb");
+    GTUtilsTaskTreeView::waitTaskFinished(os);
+    GTUtilsSequenceView::checkSequenceViewWindowIsActive(os);
```

Comment:
Usually you need either one check or another (the line before). Keep any you want, 2 checks are not needed.

Reply:
done